internal/controller/apiextensions/composite/api.go

```diff
@@ -198,16 +198,6 @@ func (f *APIRevisionFetcher) Fetch(ctx context.Context, cr resource.Composite) (
 	return AsComposition(current), nil
 }
 
-// AsComposition creates a new composition from the supplied revision. It
-// exists only as a temporary translation layer to allow us to introduce the
-// alpha CompositionRevision type with minimal changes to the XR reconciler.
-// Once CompositionRevision leaves alpha this code should be removed and the XR
-// reconciler should operate on CompositionRevisions instead.
-func AsComposition(cr *v1beta1.CompositionRevision) *v1.Composition {
```

Comment:
Is this expected?

Reply:
No, that was by mistake. I removed these changes.